routes/web.php

```diff
@@ -22,6 +22,9 @@
 
 require __DIR__.'/auth.php';
 
+Route::post('/purchase-order', [\App\Http\Controllers\PurchaseOrderController::class, "store"])->middleware(['auth']);
+Route::put('/purchase-order/{id}', [\App\Http\Controllers\PurchaseOrderController::class, "update"])->middleware(['auth']);
```

Comment:
You should be importing the fully qualified class name at the top of the file and using just the base class name here

```
use App\Http\Controllers\PurchaseOrderController;
...

Route::post('/purchase-order', [PurchaseOrderController::class, "store"])->middleware(['auth']);
...
```